File: Modul-5/soal-1/PRAK501-2410817210017-OrlandoSugian.c
#include <stdio.h>

int MaxBilangan(int a, int b, int c, int d) {
    int bilmaks = a;
    if (b > bilmaks) bilmaks = b;
    if (c > bilmaks) bilmaks = c;
    if (d > bilmaks) bilmaks = d;
    return bilmaks;
}   int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    return 0;
}

File: Modul-5/soal-1/PRAK501-2410817210017-OrlandoSugian.py
def MaxBilangan(a, b, c, d):
    bilmaks = a  
    if b > bilmaks:
        bilmaks = b 
    if c > bilmaks:
        bilmaks = c  
    if d > bilmaks:
        bilmaks = d  
    return bilmaks  
a, b, c, d = map(int, input().split())
hasil = MaxBilangan(a, b, c, d)
print(hasil)
